learn-ocaml.el

```diff
@@ -447,49 +467,76 @@ the exercise with id equal to id"
 (defun learn-ocaml-display-exercise-list ()
   ""
   (interactive)
-  (learn-ocaml-give-exercice-list
+  (learn-ocaml-give-exercise-list
    (lambda (brut-json)
      (learn-ocaml-display-exercise-list-to-wrap (elt (elt brut-json 0) 1)))))
 
 ;;
 ;; on-load management
 ;;
 
+(cl-defun learn-ocaml-init-function
+    (&key new-server-value new-token-value nickname secret callback)
+  (if (equal new-server-value nil)
```

Comment:
@leunam217 this will work, but could be simplified to `(if (not new-server-value) expr1 expr2)` or `(if new-server-value expr2 expr1)`, given that:

> In Emacs Lisp, any value that is not nil—is not the empty list—is considered true. 
− https://www.gnu.org/software/emacs/manual/html_node/eintr/nil-explained.html